lab-david/route/auth-router.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const jsonParser = require('body-parser').json();
```

Comment:
express has it's own built in body-parser that you can use, which means you need one less dependency, yay!